.gitignore

```diff
@@ -5,5 +5,6 @@
 
 node_modules/
 public/bundle.js
+public/test.js
```

Comment:
when you run `npm run build:test`, webpack both runs the tests and compiles all the code. A side effect of it running the tests is it compiles them too. I couldn't figure out how to make it not compile tests.

This is an issue that exists on master, so until we figure out how to make it not compile test.js, i figured a good thing to do was gitignore it so it stops showing up in `git status`